APIs/SentinelHub/Batch.qmd

```diff
@@ -2,7 +2,7 @@
 title: Batch Processing API
 ---
 
-**The Batch Processing API is available only for users with Copernicus Service accounts. Contact our [Copernicus Data Space Ecosystem support](https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new) to request your Copernicus Service account.**
+**The Batch Processing API is available only for users with Copernicus Service accounts.** Please refer to the Frequently Asked Question on account typology change [here](https://documentation.dataspace.copernicus.eu/FAQ.html#:~:text=How%20can%20I%20request%20to%20change%20my%20Copernicus%20user%20type%20from%20Copernicus%20General%20user%20to%20Copernicus%20Services%20user%3F) and [Submit A Request](https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new) to our Copernicus Data Space Ecosystem Support Team to request your Copernicus Service account accordingly.
```

Comment:
what is this `#:~:text=How%20can%20I%20request%20to%20chan...`  stuff? Is that a copy-paste gone wrong or is it intended? In case of the latter: that doesn't look like a stable (long term) reference